How-To/Connect-to-AWS-Resources.md

```diff
@@ -122,16 +122,13 @@ As with R, enabling [unload](https://laughingman7743.github.io/PyAthena/pandas.h
     import pandas
     import pyarrow
     from pyathena import connect
-    from pyathena.pandas.util import as_pandas
-    from pyathena.pandas.cursor import PandasCursor
+    from pyathena.arrow.cursor import ArrowCursor
 
     # Connect to Athena
     cursor = connect(
-        # We add '+ "/"' to the end of the line below because enabling unload
-        # requires that the staging directory end with a slash
-        s3_staging_dir=os.getenv("AWS_ATHENA_S3_STAGING_DIR") + "/",
+        s3_staging_dir=os.getenv("AWS_ATHENA_S3_STAGING_DIR"),
```

Comment:
This trailing '/' problem seems to have vanished after changing to `ArrowCursor`